src/Symfony/Component/HttpFoundation/Session/Attribute/NamespacedAttributeBag.php

```diff
@@ -43,7 +43,7 @@ public function __construct($storageKey = '_sf2_attributes', $namespaceCharacter
      */
     public function has($name)
     {
-        $attributes = $this->resolveAttributePath($name);
+        $attributes = &$this->resolveAttributePath($name);
```

Comment:
:-1: for this one and the same one below: the array_key_exists check that is called a few lines below will now trigger a new ref mismatch. Unfortunately, their is no easy ref-safe way to check if a key exists and is not null.